gcloud-java-dns/src/test/java/com/google/gcloud/dns/ManagedZoneInfoTest.java

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.dns;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import com.google.common.collect.Lists;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ManagedZoneInfoTest {
+
+  private static final String NAME = "mz-example.com";
+  private static final Long ID = 123L;
+  private static final Long CREATION_TIME_MILLIS = 1123468321321L;
+  private static final String DNS_NAME = "example.com.";
+  private static final String DESCRIPTION = "description for the zone";
+  private static final String NAME_SERVER_SET = "some set";
+  private static final String NS1 = "name server 1";
+  private static final String NS2 = "name server 2";
+  private static final String NS3 = "name server 3";
+  private static List<String> nameServers = new LinkedList<>();
+  private static ManagedZoneInfo info;
+
+  @BeforeClass
+  public static void setUp() {
+    nameServers.add(NS1);
```

Comment:
If you were not modifying it in the tests than that is my mistake. I would than switch it  back to static and final. Also, you don't need the `@BeforeClass` setup in this case as both nameServers could be assigned in the declaration. For `nameServers` use `ImmutableList.of(NS1, NS2, NS3)`.